.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI/CD
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: 8
+
+      - name: Install libxml2-utils and rpm
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libxml2-utils rpm
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up Maven
+        uses: actions/setup-java@v2
+        with:
+          java-version: 8
+          maven-version: 3.6.3
+
+      - name: Generate Graylog resources
+        run: |
+          export GRAYLOG_VERSION=$(xmllint --xpath '/*[local-name()="project"]/*[local-name()="properties"]/*[local-name()="graylog.version"]/text()' pom.xml)
+          echo "Checking out Graylog ${GRAYLOG_VERSION}"
+          git clone --depth 1 --branch "${GRAYLOG_VERSION}" https://github.com/Graylog2/graylog2-server.git ../graylog2-server
+          cd ../graylog2-server
+          mvn generate-resources -pl graylog2-server -B -V
+
+      - name: Build and package
+        run: |
+          cd ${{ github.workspace }}
+          mvn jdeb:jdeb
+          mvn rpm:rpm
+          rm -f target/original-*.jar
+
+      - name: Set up Git
+        run: |
+          git config --global user.name "Your Name"
+          git config --global user.email "your-email@example.com"
+
+      - name: Deploy release to GitHub
+        uses: actions/upload-release-asset@v1
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: |
+            target/*.jar
+            target/*.deb
+            target/*.rpm
+          asset_name: ${{ github.event.release.tag_name }}
+          asset_content_type: application/java-archive
+        env:
+          GITHUB_TOKEN: ${{ secrets.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  }}
```

Comment:
I guess I have to register a token as a GitHub secret, right?

Reply:
Yes @hidapple .You will have to register a token as a GitHub secret.

Reply:
@GP1812 Sorry for being too late to reply to you. I've just registered PAT as `GRAYLOG_PLUGIN_TEAMS_PAT`. Can you update this line to refer the secret name?